OSDOCS-14370: Documented the 4.17.26 z-stream release notes (#92433)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

[source,terminal]
----
$ oc adm release info 4.17.26 --pullspecs
----

[id="ocp-4-17-26-known-issues_{context}"]
==== Known issues

* There is a known issue when the grandmaster clock (T-GM) transitions to the Locked state too soon. This happens before the Digital Phase-Locked Loop (DPLL) completes its transition to the `Locked-HO-Acquired` state, and after the Global Navigation Satellite Systems (GNSS) time source is restored.
(link:https://issues.redhat.com/browse/OCPBUGS-54534[*OCPBUGS-54534*]) 

[id="ocp-4-17-26-bug-fixes_{context}"]
==== Bug fixes

* Previously, the Single Root I/O Virtualization (SR-IOV) virtual function (VF) did not revert any unexpected value changes to the maximum transmission unit (MTU) value when a pod was deleted. This issue occurred if the is the application inside the pod had its MTU value changed; in turn, the pod would also have its MTU value changed. With this release, the SR-IOV Container Network Interface (CNI) now reverts any unexpected MTU value changes to the original value so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-54392[*OCPBUGS-54392*])

> * Previously, when you attempted to create a validating webhook for a resource that is managed by the `oauth` API server, the validating webhook was not created. This issue occurred because of a communication issue with the `oauth` API server and the data plane. With this release, a Konnectivity proxy sidecar has been added to bridge communications between the `oauth` API server and the data plane so that you can now create a validating webhook for any resource that the `oauth` API server manages.  (link:https://issues.redhat.com/browse/OCPBUGS-54841[*OCPBUGS-54841*])

Author: :heavy_check_mark:

* Previously, virtual machines (VMs) in a cluster that ran on {azure-first} failed because the attached network interface controller (NIC) was in a `ProvisioningFailed` state. With this release, the Machine API controller now checks the provisioning status of a NIC and refreshes the VMs on a regular basis to prevent this issue. (link:https://issues.redhat.com/browse/OCPBUGS-54393[*OCPBUGS-54393*]) 

> * Previously, the Installer panicked if it attempted to retrieve {gcp-full} tags over an unstable network, or when it could not reach the GCP server. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-51210[*OCPBUGS-51210*]) 

Reviewer: The word "panicked" seems a bit anthropomorphic here. Would a word like "malfunctioned" work instead?

> * Previously, a User Datagram Protocol (UDP) packet that was larger than the maximum transmission unit (MTU) value set for the cluster, could not be sent to the endpoint of the packet by using a service. With this release, the pod IP address is used instead of the service IP address regardless of the packet size, so that the UDP packet can be sent to the endpoint.(link:https://issues.redhat.com/browse/OCPBUGS-50579[*OCPBUGS-50579*])

Reviewer: ```suggestion
* Previously, a User Datagram Protocol (UDP) packet that was larger than the maximum transmission unit (MTU) value set for the cluster could not be sent to the endpoint of the packet by using a service. With this release, the pod IP address is used instead of the service IP address regardless of the packet size, so that the UDP packet can be sent to the endpoint. (link:https://issues.redhat.com/browse/OCPBUGS-50579[*OCPBUGS-50579*])
```

// SME review

> * Previously, you could add white space to {aws-first} tag names but the installation program did not support them. This situation resulted in the installation program outputting an `ERROR failed to fetch Metadata` message. With this release, the regular expression for {aws-short} tags now validates any tag name that has white space so that the installation program accepts these tags and no longer outputs an error because of white space. (link:https://issues.redhat.com/browse/OCPBUGS-46375[*OCPBUGS-46375*])

Reviewer: ```suggestion
* Previously, you could add white space to {aws-first} tag names, but the installation program did not support them. As a result, the installation program produced an `ERROR failed to fetch Metadata` message. With this release, the regular expression for {aws-short} tags validates any tag name that has white space so that the installation program accepts these tags. (link:https://issues.redhat.com/browse/OCPBUGS-46375[*OCPBUGS-46375*])
```
IBM Style warns against using "output" as a verb: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#output
